webapp/src/main/webapp/news-publish-targets-management/components/NewsPublishTargetsManagementDrawer.vue

```diff
@@ -159,6 +159,7 @@ export default {
     originalTargetName: '',
     saveMode: 'creationMode',
     permissions: [],
+    permissionDisabled: true,
```

Comment:
why do we need permissionDisabled ?

Reply:
> why do we need permissionDisabled ?

To detect if there is any change for the list of permissions in case of edit to enable the button update.

Reply:
IMHO it is should be better !permissionsUpdated

Reply:
> IMHO it is should be better !permissionsUpdated

Done !